src/expect.js

```diff
@@ -1,88 +1,61 @@
-const { prettyPrint, areSameObjectProperties } = require('./utilities');
+const { prettyPrint, areSameObjectProperties } = require("./utilities");
 
 // expect(something).toBe(something2);
 
-function toBe(actualValue) {
-  return function (expectedValue) {
-    if (Object.is(expectedValue, actualValue)) {
-      console.log('✅');
-    } else {
-      console.error(
-        `❌ expected ${prettyPrint(actualValue)} to equal ${prettyPrint(
-          expectedValue,
-        )}`,
-      );
-    }
-  };
-}
-
-function toEqual(actualValue) {
-  return function (expectedValue) {
-    if (typeof expectedValue === 'object') {
-      if (areSameObjectProperties(expectedValue, actualValue)) {
-        console.log('✅');
-      } else {
-        console.error(
-          `❌ expected ${prettyPrint(actualValue)} to equal ${prettyPrint(
-            expectedValue,
-          )}`,
-        );
-      }
-    } else {
-      if (expectedValue == actualValue) {
-        console.log('✅');
-      } else {
-        console.error(
-          `❌ expected ${prettyPrint(actualValue)} to equal ${prettyPrint(
-            expectedValue,
-          )}`,
-        );
-      }
-    }
-  };
-}
-
 function expect(actualValue) {
-  return {
-    not: {
+  const functions = (invert) => {
```

Comment:
Refactor (blocking): `functions` is too generic of a name. `matchers` is more specific.